lingetic-spring-backend/Dockerfile

```
@@ -1,10 +1,30 @@
+RUN yum update -y && yum install -y findutils
+
+COPY gradlew .
+COPY gradle gradle
+COPY build.gradle.kts .
+COPY settings.gradle.kts .
+
+COPY config/application.properties config/application.properties
+
+COPY src/main src/main
+
+RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
```

Hadolint (2.12.0)

[warning] Quote this to prevent word splitting.

(SC2046)